frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionsForm.tsx

```diff
@@ -262,6 +308,7 @@ const mapStateToProps = ({ k8s }) => {
 
   return {
     hasSriovNetNodePolicyCRD: !kindsInFlight && !!k8sModels.get(SriovNetworkNodePolicyModel.kind),
+    hasHyperConvergedCRD: !kindsInFlight && !!k8sModels.get('HyperConverged'),
```

Comment:
dont we want to create model for `HyperConverged` CRD ?

Reply:
We're not actually using the `HyperConverged` CRD aside from testing that it exists, so I wasn't sure if I should create one. I went ahead and made it. It's easy enough to remove if we decide it's unnecessary.